<launch>
  <arg name="rqt" default="false"/>
  <arg name="sonar" default="true"/>

  <!-- model: M750d / M1200d -->
  <arg name="model" default="M750d"/>
  <arg name="model_2" default="M1200d"/>

  <!-- Start the sonar reading -->
  <node if="$(arg sonar)" name="sonar_oculus_node_$(arg model)" pkg="sonar_oculus" type="sonar_oculus_node" output="screen">
    <param name="model" value="$(arg model)"/>
    <param name="frame" type="string" value="/$(arg model)"/>
  </node>

  <!-- Start the sonar reading -->
  <node if="$(arg sonar)" name="sonar_oculus_node_$(arg model_2)" pkg="sonar_oculus" type="sonar_oculus_node" output="screen">
    <param name="model" value="$(arg model_2)"/>
    <param name="frame" type="string" value="/$(arg model_2)"/>
  </node>


</launch>
